server/cluster/coordinator.go

```diff
@@ -284,7 +284,12 @@ func (c *coordinator) LoadPlugin(pluginPath string, ch chan string) {
 		return
 	}
 	log.Info("create scheduler", zap.String("scheduler-name", s.GetName()))
-	if err = c.addUserScheduler(s); err != nil {
+	if _, ok := c.schedulers[s.GetName()]; ok {
```

Comment:
I think if we already have a scheduler with the same name, the load process should report error and exit instead of removing the original one?